google/ggrc-core
# GGRC-6413 Add "Change Request" option on My Tasks page

# Issue description

This feature should be turned on/off at the moment of deployment with the help of deployment variable.
If the feature is turned on, the "Change Request" option should be displayed under 3 dot on My Tasks page.
On clicking "Change Request" option, the link should be opened in a new tab: 

# Steps to test the changes

1. Go to My Task page
2. Click 3 dots menu

Expected result: "Change Request" option should be available if appropriate `CHANGE_REQUEST_URL` is set.

From BE:
1. Change default value for CHANGE_REQUEST_URL env variable in `ggrc/settings/default.py`;
2. Run application locally;
3. Login into app as admin;
4. Open development tools/console;
5. Print GGRC.config. CHANGE_REQUEST_URL variable value.

# Solution description

FE:
Added `change-request-link` component.

BE:
Added `CHANGE_REQUEST_URL` variable. 

# Sanity checklist




# PR Review checklist
